libs/payments/ui/src/lib/nestapp/validators/GetSubManPageContentActionResult.ts

```diff
@@ -0,0 +1,45 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { Type } from 'class-transformer';
+import {
+  IsArray,
+  IsNumber,
+  IsOptional,
+  IsString,
+  ValidateNested,
+} from 'class-validator';
+import type { PaymentProvidersType } from '@fxa/payments/customer';
+import { StripeSubscription } from '@fxa/payments/stripe';
+
+class DefaultPaymentMethod {
+  @IsString()
+  type!: PaymentProvidersType;
+
+  @IsOptional()
+  @IsString()
+  brand?: string;
+
+  @IsOptional()
+  @IsString()
+  last4?: string;
+
+  @IsOptional()
+  @IsNumber()
+  expMonth?: number;
+
+  @IsOptional()
+  @IsNumber()
+  expYear?: number;
+}
+
+export class GetSubManPageContentActionResult {
+  @IsOptional()
+  @ValidateNested()
+  @Type(() => DefaultPaymentMethod)
+  defaultPaymentMethod?: DefaultPaymentMethod;
+
+  @IsArray()
+  subscriptions!: StripeSubscription[];
```

Comment:
issue: This needs a class validator. I also wouldn't expect the entire StripeSubscriptions to part of the result.

suggestion: only return the values of the subscription needed by the frontend.